components/actions_menu/index.ts

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {ComponentProps} from 'react';
+import {connect} from 'react-redux';
+import {ActionCreatorsMapObject, bindActionCreators, Dispatch} from 'redux';
+
+import {getCurrentUser, getCurrentUserId} from 'mattermost-redux/selectors/entities/users';
+import {getCurrentTeamId, getCurrentTeam} from 'mattermost-redux/selectors/entities/teams';
+import {appsEnabled, makeGetPostOptionBinding} from 'mattermost-redux/selectors/entities/apps';
+
+import {AppBindingLocations} from 'mattermost-redux/constants/apps';
+
+import {isSystemAdmin} from 'utils/utils';
+import {isSystemMessage} from 'mattermost-redux/utils/post_utils';
+import {isCombinedUserActivityPost} from 'mattermost-redux/utils/post_list';
+
+import {ActionFunc, GenericAction} from 'mattermost-redux/types/actions';
+import {AppBinding} from 'mattermost-redux/types/apps';
+import {Post} from 'mattermost-redux/types/posts';
+import {DoAppCall, PostEphemeralCallResponseForPost} from 'types/apps';
+import {GlobalState} from 'types/store';
+
+import {openModal} from 'actions/views/modals';
+import {doAppCall, makeFetchBindings, postEphemeralCallResponseForPost} from 'actions/apps';
+
+import ActionsMenu from './actions_menu';
+
+type Props = {
+    post: Post;
+    handleCardClick?: (post: Post) => void;
+    handleDropdownOpened: (open: boolean) => void;
+    isMenuOpen: boolean;
+    location?: ComponentProps<typeof ActionsMenu>['location'];
+};
+
+const emptyBindings: AppBinding[] = [];
+
+const getPostOptionBinding = makeGetPostOptionBinding();
+
+const fetchBindings = makeFetchBindings(AppBindingLocations.POST_MENU_ITEM);
+
+function mapStateToProps(state: GlobalState, ownProps: Props) {
+    const {post} = ownProps;
+
+    const userId = getCurrentUserId(state);
+    const currentTeam = getCurrentTeam(state) || {};
```

Comment:
I think this could be removed, since we also have in the return object `teamId: getCurrentTeamId(state)`

Reply:
done 3486130